src/service/nes/modules/sbom.ts

```diff
@@ -1,77 +1,70 @@
-import { gql } from "@apollo/client/core"
+import { gql } from '@apollo/client/core/core.cjs';
 
-import { log } from '../../../utils/log.util';
-import { SbomMap } from '../../eol/eol.types';
-import { ApolloHelper } from "../nes.client"
+import { log } from '../../../utils/log.util.ts';
+import type { SbomMap } from '../../eol/eol.types.ts';
+import type { ApolloHelper } from '../nes.client.ts';
 
-export const buildScanResult =
-  (scan: ScanResponseReport): ScanResult => ({
-    components: Object.fromEntries(scan.components.map((c) => [c.purl, c])),
-    message: scan.message,
-    success: true
-  })
+export const buildScanResult = (scan: ScanResponseReport): ScanResult => ({
+  components: Object.fromEntries(scan.components.map((c) => [c.purl, c])),
+  message: scan.message,
+  success: true,
+});
 
-export const SbomScanner = (client: ApolloHelper) =>
-  async (sbom: SbomMap ): Promise<ScanResult> => {
+export const SbomScanner =
+  (client: ApolloHelper) =>
+  async (sbom: SbomMap): Promise<ScanResult> => {
+    const input: ScanInput = { components: sbom.purls, type: 'SBOM' };
+    const res = await client.mutate<ScanResponse, { input: ScanInput }>(M_SCAN.gql, { input });
 
-    const input: ScanInput = { components: sbom.purls, type: 'SBOM' }
-    const res = await client.mutate<ScanResponse, { input: ScanInput }>(M_SCAN.gql, { input })
-
-    const scan = res.data?.insights?.scan?.eol
+    const scan = res.data?.insights?.scan?.eol;
     if (!scan?.success) {
+      log.info('failed scan %o', scan || {});
+      log.warn('scan failed');
```

Comment:
warn doesn't take args or something weird. we should figure out a better logging strategy (not for acceptance. just planting seed)